### DISCUSSION TITLE
为什么文件夹下载不行了呢

### DISCUSSION THREAD

**Discussion by arbog2** [AlistGo/alist#955] [Q&A]
正在下载
正在下载文件，请勿关闭或刷新页面
一直显示上面这个

环境是 Debian下部署的，然后同服务器下nginx 开通了https 反向代理到127.0.0.1：5244

**Comment by xhofe** [comment]
请提供能复现此问题的链接

**Reply by arbog2** [reply]
https://f.sytzzx.com

**Reply by xhofe** [reply]
检查你的反向代理配置。
